src/toast/ToastController.tsx

```diff
@@ -2,10 +2,29 @@ import React from "react";
 import { useTimeout } from "@chakra-ui/hooks";
```

Comment:
Can we use their hooks package OR should we also localize this ?

Reply:
Can we reexport form chakra/hook from our newly created "hooks" folder 🤔

Reply:
Depends on if we are using chakra hooks anywhere else, if this is the only time we are using the package we can extract it out

Reply:
We are using many hooks from chakra.

Reply:
Then i think we can keep it, since it would be well tested & updated.

Reply:
Okay, okay !!

Reply:
Yeah